Makefile

```diff
@@ -6,6 +6,9 @@ endif
 IMAGE=$(NAME):$(VERSION)
 IMAGE_LATEST=$(IMAGE)-latest
 ODOO_URL=https://github.com/odoo/odoo/archive/$(VERSION).tar.gz
+ifeq (VERSION, "11.0")
```

Comment:
To remove once V11 is released

Reply:
you can remove it, branch 11.0 has been created:

https://github.com/odoo/odoo/tree/11.0